raiden/utils/__init__.py

```diff
@@ -53,6 +53,15 @@
 GLOBAL_CTX = secp256k1.lib.secp256k1_context_create(secp256k1.ALL_FLAGS)
 
 
+def safe_address_decode(address):
+    try:
+        address = address.decode('hex')
+    except TypeError:
```

Comment:
Cool, makes sense. Just had different expectations due to the `safe` part of the function name but I guess it's perfectly fine.